pkg/pulumiyaml/diags/diags_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2023, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package diags
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNonExistentFieldFormatter_fieldsName(t *testing.T) {
```

Comment:
Nit: Designating a subtest via `_` exists no where else in this code base. Can we stick to the same naming convention as everywhere else?
```suggestion
func TestNonExistentFieldFormatterFieldsName(t *testing.T) {
```